### PR TITLE
Unify python file header

### PR DESCRIPTION
**Pull request by mmacata**
During development some python file headers got mixed up. This PR unifies them while considering the license.

### CODE REVIEW

File: actinia_gdi/api/gdi_processing.py [hunk 1]
```diff
@@ -19,11 +36,17 @@
...
-#######
 
-"""
+
 Extension of actinia_cores ephemeral_processing_with_export AsyncEphemeralExportResource to include process chain templates
 """
+
+__license__ = "Apache-2.0"
+__author__ = "Anika Bettge, Sören Gebbert"
+__copyright__ = "Copyright 2016-2018, Sören Gebbert and mundialis GmbH & Co. KG"
```
Comment (anikaweinmann) [inline actinia_gdi/api/gdi_processing.py]: I thing the gdi_processing.py is also from 2019
```suggestion
__copyright__ = "Copyright 2016-2019, Sören Gebbert and mundialis GmbH & Co. KG"
```

**Review by anikaweinmann — APPROVED** [review]
Thank you

File: actinia_gdi/api/gdi_processing.py [hunk 2]
```diff
@@ -1,5 +1,22 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
-#######
+"""
+Copyright (c) 2018-present mundialis GmbH & Co. KG
```
Comment (neteler) [inline actinia_gdi/api/gdi_processing.py]:
```suggestion
Copyright (c) 2018-2021 mundialis GmbH & Co. KG
```

File: actinia_gdi/api/gmodules/actinia.py [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-#######
-
 """
+Copyright (c) 2018-present mundialis GmbH & Co. KG
```
Comment (neteler) [inline actinia_gdi/api/gmodules/actinia.py]:
```suggestion
Copyright (c) 2018-2021 mundialis GmbH & Co. KG
```

File: actinia_gdi/api/gmodules/combined.py [hunk 4]
```diff
@@ -1,33 +1,33 @@
...
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-#######
-
 """
+Copyright (c) 2018-present mundialis GmbH & Co. KG
```
Comment (neteler) [inline actinia_gdi/api/gmodules/combined.py]:
```suggestion
Copyright (c) 2018-2021 mundialis GmbH & Co. KG
```

File: actinia_gdi/api/gdi_processing.py (hunk 2 above)
Comment (neteler) [reply actinia_gdi/api/gdi_processing.py]: (I don't think that "present" is legally reasonable)

File: actinia_gdi/api/gmodules/grass.py [hunk 5]
```diff
@@ -1,34 +1,33 @@
...
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-#######
-
 """
-Module management
+Copyright (c) 2018-present mundialis GmbH & Co. KG
```
Comment (neteler) [inline actinia_gdi/api/gmodules/grass.py]:
```suggestion
Copyright (c) 2018-2021 mundialis GmbH & Co. KG
```

File: actinia_gdi/core/gmodulesActinia.py [hunk 6]
```diff
@@ -1,30 +1,30 @@
...
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-#######
-
 """
-Module management related to process chain templates
+Copyright (c) 2018-present mundialis GmbH & Co. KG
```
Comment (neteler) [inline actinia_gdi/core/gmodulesActinia.py]:
```suggestion
Copyright (c) 2018-2021 mundialis GmbH & Co. KG
```

File: actinia_gdi/core/gmodulesGrass.py [hunk 7]
```diff
@@ -1,30 +1,30 @@
...
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-#######
-
 """
-Module management related to GRASS modules
+Copyright (c) 2018-present mundialis GmbH & Co. KG
```
Comment (neteler) [inline actinia_gdi/core/gmodulesGrass.py]:
```suggestion
Copyright (c) 2018-2021 mundialis GmbH & Co. KG
```

File: actinia_gdi/core/gmodulesParser.py [hunk 8]
```diff
@@ -1,30 +1,30 @@
...
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-#######
-
 """
-Module management to parser GRASS xml response
+Copyright (c) 2018-present mundialis GmbH & Co. KG
```
Comment (neteler) [inline actinia_gdi/core/gmodulesParser.py]:
```suggestion
Copyright (c) 2018-2021 mundialis GmbH & Co. KG
```

File: actinia_gdi/core/gmodulesProcessor.py [hunk 9]
```diff
@@ -1,5 +1,22 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
-#######
+"""
+Copyright (c) 2018-present mundialis GmbH & Co. KG
```
Comment (neteler) [inline actinia_gdi/core/gmodulesProcessor.py]:
```suggestion
Copyright (c) 2018-2021 mundialis GmbH & Co. KG
```

File: actinia_gdi/resources/logging.py [hunk 10]
```diff
@@ -1,3 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Copyright (c) 2018-present mundialis GmbH & Co. KG
```
Comment (neteler) [inline actinia_gdi/resources/logging.py]:
```suggestion
Copyright (c) 2018-2021 mundialis GmbH & Co. KG
```

**Review by neteler — APPROVED** [review]

**Pull request merged**